src/apps/service/apps/jobs/submissions.clj

```diff
@@ -45,7 +45,7 @@
   [user input-paths-by-id]
   (->> (flatten (vals input-paths-by-id))
        (remove string/blank?)
-       (get-file-stats user)
+       (#(get-file-stats user % :filter-include "infoType,path,file-size"))
```

Comment:
This one's the complex bit of filtering. `infoType` is used just below. `path` and `file-size` are both used by `validate-path-list-stats`. After validating, it's passed to `extract-ht-param-ids` which seems to only use the path (via `(map :path path-list-stats)`). Finally it's used with another `(map :path ` in submit-batch-job itself.